src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/LinuxNetworkInterface.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
```

Comment:
Nit: most of these are unused. VS has a nice feature that shows you which ones. Also, ensure that they are sorted in all files.
(Will stop commenting on this.)